src/createLanguageClient.ts

```diff
@@ -85,19 +85,19 @@ async function openConnection (url: URL | string, errorHandler: ConnectionErrorH
 const RETRY_DELAY = 3000
 class CGLSPConnectionProvider implements IConnectionProvider {
   private readonly serverUrl: string
-  private readonly language: string
+  private readonly id: string
```

Comment:
nit: I'm wondering if the type could be an enum of possible values instead of a string. What do you think?

Reply:
It's a good idea, I already though about it

Reply:
https://github.com/CodinGame/monaco-languageclient-wrapper/pull/4/commits/67551ffb2b457526b0fbd6382388b22c647e75ee